### PR TITLE
docs: add comma in list

### PR DESCRIPTION
**Pull request by leoalves100**

### CODE REVIEW

File: versioned_docs/version-0.34.0/providers/aws/authentication.mdx [hunk 1]
```diff
@@ -87,7 +87,7 @@ data "aws_iam_policy_document" "policy" {
       "ec2:DescribeLaunchTemplates",
       "ec2:GetEbsEncryptionByDefault",
       "ecr:DescribeRepositories",
-      "ecr:GetRepositoryPolicy"
+      "ecr:GetRepositoryPolicy",
```
Comment (wbeuil) [inline versioned_docs/version-0.34.0/providers/aws/authentication.mdx:90]: Good catch !

File: versioned_docs/version-0.34.0/providers/aws/authentication.mdx [hunk 2]
```diff
@@ -170,7 +170,7 @@ data "aws_iam_policy_document" "policy" {
       "application-autoscaling:DescribeScheduledActions",
       "autoscaling:DescribeLaunchConfigurations",
       "elasticloadbalancing:DescribeLoadBalancers",
-      "elasticloadbalancing:DescribeListeners"
+      "elasticloadbalancing:DescribeListeners",
```
Comment (wbeuil) [inline versioned_docs/version-0.34.0/providers/aws/authentication.mdx:173]: The last policy does not need a comma, you can remove this one and the other one.

File: versioned_docs/version-0.34.0/providers/aws/authentication.mdx [hunk 3]
```diff
@@ -358,7 +358,7 @@ As AWS documentation recommends, the below policy is granting only the permissio
                 "application-autoscaling:DescribeScheduledActions",
                 "autoscaling:DescribeLaunchConfigurations",
                 "elasticloadbalancing:DescribeLoadBalancers",
-                "elasticloadbalancing:DescribeListeners"
+                "elasticloadbalancing:DescribeListeners",
```
Comment (eliecharra) [inline versioned_docs/version-0.34.0/providers/aws/authentication.mdx:361]: The latest element of an array should not have a comma

File: versioned_docs/version-0.34.0/providers/aws/authentication.mdx (hunk 2 above)
Comment (eliecharra) [inline versioned_docs/version-0.34.0/providers/aws/authentication.mdx:173]: The latest element of an array should not have a comma

**Review by eliecharra — CHANGES_REQUESTED** [review]
Thanks for the catch on `ecr:GetRepositoryPolicy`, but for the latest elements we should not have a comment, could you fix that ?

Thanks 😃

**Review by wbeuil — CHANGES_REQUESTED** [review]

**Comment by wbeuil** [comment]
Please refer to the first reviewer :)

**Comment by wbeuil** [comment]
I'm gonna close this one to only merge [the other one](https://github.com/snyk/driftctl-docs/pull/245)

**Pull request closed without merging**